src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -553,7 +645,222 @@ unittest
     Test.test!("==")(isRelativeTime("2h"), true);
     Test.test!("==")(isRelativeTime("1d"), true);
     Test.test!("==")(isRelativeTime("2d"), true);
+    Test.test!("==")(isRelativeTime("2d/d"), true);
 
     Test.test!("==")(isRelativeTime("1x"), false);
     Test.test!("==")(isRelativeTime("2xm"), false);
 }
+
+/******************************************************************************
+
+    Check if a char value is a valid time unit
+
+    Params:
+        unit = the char to be checked
+
+    Returns:
+        true if the unit is valid
+
+******************************************************************************/
+
+private bool isValidTimeUnit ( char unit )
+{
```

Comment:
Taking out the `unit == 's' || unit == 'm' || unit == 'h' || unit == 'd'` code and putting it a separate function `isValidTimeUnit` can be in its own commit (before this commit)

Reply:
Should I go in that many details with the commits?